### DISCUSSION TITLE
Welcome to Microsoft-Future_Ready_Talent-Internship Discussions!

### DISCUSSION THREAD

**Discussion by TheSuranaverse** [Neklaustares-tPtwP/Microsoft-Future_Ready_Talent-Internship#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kittu-2208** [comment]
Hello, l'm Varsha. I have enrolled in future ready talent Microsoft internship. I am unable to access the self learning courses. Please help me

**Reply by Tarunkumar1041** [reply]
hi, I am Tarun all you have to do is update your profiles like github, aicte, future prime, mslearn and linkedin in the orientation then you will be able to access self learning

**Reply by Sejal-1234** [reply]
hi, i  sejal. i am not unable to access  self learning program  .please help me

**Comment by ayesha8433** [comment]
hello! even though im completing my modules, it's not getting updated, it's showinh in-progress, i have entirely completed the module including tests
